src/android/PurchasesPlugin.java

```diff
@@ -416,6 +416,24 @@ private void presentCodeRedemptionSheet(CallbackContext callbackContext) {
         callbackContext.success();
     }
 
+    @PluginAction(thread = ExecutionThread.WORKER, actionName = "beginRefundRequestForActiveEntitlement")
+    private void beginRefundRequestForActiveEntitlement(CallbackContext callbackContext) {
+        // noop
+        callbackContext.success();
+    }
+
+    @PluginAction(thread = ExecutionThread.WORKER, actionName = "beginRefundRequestForEntitlementId")
+    private void beginRefundRequestForEntitlementId(CallbackContext callbackContext) {
+        // noop
+        callbackContext.success();
+    }
+
+    @PluginAction(thread = ExecutionThread.WORKER, actionName = "beginRefundRequestForProductId")
+    private void beginRefundRequestForProductId(CallbackContext callbackContext) {
+        // noop
+        callbackContext.success();
```

Comment:
Hmm should we error out if calling these in android?

Reply:
we don't error out in any other method that's only available in iOS, so I don't think we should have some functions that error and some that don't. I think we should update all of them at once and do error out though, but that's a breaking change so we should do that in the next major probably. Do you agree? If you strongly feel we should error here, I will add it